geoips/schema/product_defaults/bases/windbarb_plotters.yaml

```diff
@@ -0,0 +1,40 @@
+$id: product_defaults.bases.windbarb_plotter
```

Comment:
Is it a problem that the YAML filename is "windbarb_plotters" and id is "windbarb_plotter"?

Reply:
I don't think it would have mattered, but I changed the file name for consistency. It is now `windbarb_plotter.yaml`.